anime_downloader/util.py

```diff
@@ -46,15 +46,17 @@ def search(query):
         logging.error(click.style(str(e), fg='red'))
         sys.exit(1)
     click.echo(format_search_results(search_results))
-
+    if not search_results: 
+        print('No such Anime found')
+        sys.exit(1)
     val = click.prompt('Enter the anime no: ', type=int, default=1)
 
     try:
         url = search_results[val-1].url
         title = search_results[val-1].title
     except IndexError:
-        logging.error('Only maximum of 30 search results are allowed.'
-                      ' Please input a number less than 31')
+        logging.error('Only maximum of {} search results are allowed.'
+                      ' Please input a number less than {}'.format(val-1,val))
```

Comment:
Instead of `val-1`, use `len(search_results)`

Reply:
Done